### DISCUSSION TITLE
Fabric - no such file or directory

### DISCUSSION THREAD

**Discussion by LukePenkava** [invertase/react-native-firebase#4353] [Q&A] [answered]
Hi, I am using RN 0.63.3 and RNF and got this error. I had suspicion it had something to do with Crashlytics and received suggestion it could be related to RNF, but i am not sure. I cant find almost anything on this error, so just wanted to ask. Btw i am doing update from react-native 0.61 to 0.63.3 along with other things since last November.

    `ios/Pods/Fabric/run: No such file or directory`

"react-native": "0.63.3",
"@react-native-firebase/analytics": "^7.6.7",
"@react-native-firebase/app": "^8.4.5",
"@react-native-firebase/auth": "^9.2.7",
"@react-native-firebase/crashlytics": "^8.4.8",
"@react-native-firebase/database": "^7.5.10",
"@react-native-firebase/messaging": "^7.8.10",

Thanks for any help.

**Comment by mikehardy** [comment, marked as answer]
Hi there! Definitely related to Firebase, and specifically to the RNFB "Fabric Crashlytics -> Firebase Crashlytics" migration.

Two thoughts: 

1- that script is used to upload dSYMs which are effectively worthless in the bitcode era - https://stackoverflow.com/a/55796619/9910298

2- if you want to continue to upload your locally created dSYMs (perhaps for checking a debug stack trace, it can have limited purpose at least) you need to call the new Crashlytics script, as mentioned in the setup: https://github.com/invertase/react-native-firebase/commit/cad58e178b43dea461e17fa4a0a3fecd507ba68a#diff-3bd938c98a87fa81f16efc012907ffb0

**Reply by LukePenkava** [reply]
@mikehardy Thanks for quick reply. Is it ok to remove Crashlytics as a solution?

**Reply by LukePenkava** [reply]
@mikehardy Unfortunately it did not help. I removed the Crashlytics module, but getting same error.

**Reply by mikehardy** [reply]
that's not the right course of action as you can see it's ineffective. I mean, remove crashlytics if you don't need it, but either way if you integrated it before, then in your Xcode project config you are referencing an old script, and you need to either change the reference to the new script, or you need to remove that script from your build phases entirely. The rnfirebase.io crashlytics documents show where it is

**Reply by LukePenkava** [reply]
@mikehardy I can remove it or keep it in, whatever is the easiest. I have not realy used Crashlytics as far as i know. I am super rusty on RN and on the project as i didnt touch it for some time and just need to update it.

I am not realy importing Crashlytics anywhere in any component. All i could find in my project was bunch of README.md files and then in these:

project.pbxproj
Firebase.h
RNFBCrashltics.podspec.json

Again, i am totaly fine with keeping Crashlytics or removing it, whatever is the easiest solution to this error as i got stuck on it. 

Thanks

**Reply by mikehardy** [reply]
read the rnfirebase.io crashlytics docs. Find the part about adding this script. Go to that part of your Xcode project, in Xcode. Either alter it if you want to keep crashlytics, or remove it if you want to remove crashlytics.

**Reply by LukePenkava** [reply]
@mikehardy Thanks, i have removed the line in Build Phases in Run Script. Its working now.
